ddtrace/settings/config.py

```diff
@@ -569,6 +569,7 @@ def __init__(self):
         self._llmobs_sample_rate = float(os.getenv("DD_LLMOBS_SAMPLE_RATE", 1.0))
         self._llmobs_ml_app = os.getenv("DD_LLMOBS_ML_APP")
         self._llmobs_agentless_enabled = asbool(os.getenv("DD_LLMOBS_AGENTLESS_ENABLED", False))
+        self._llmobs_ragas_faithfulness_enabled = asbool(os.getenv("_DD_LLMOBS_RAGAS_FAITHFULNESS_ENABLED", False))
```

Comment:
I don't think I love having a boolean flag for every single RAGAS metric. What if we were to make this a comma-separated string option that specifies which metrics to enable? i.e. `DD_LLMOBS_RAGAS_METRICS="faithfulness"`

Also, do you think it might be a better idea to keep this a simple os environment variable instead of declaring it as a config option until we're more sure about the overall API? Changing/removing config options are not simple (technically it is changing the public API)

Reply:
I like both of those ideas, we can
1. remove the config option
2. only read from `DD_LLMOBS_RAGAS_METRICS="faithfulness,coherence,etc.."`